civihr_default_theme/scss/components/_leave-calendar.scss

```diff
@@ -0,0 +1,122 @@
+$chr-leave-calendar-cell-height:                  35px;
+$chr-leave-calendar-cell-font-size:               10px;
+$chr-leave-calendar-item-requested-border-color:  #c7cbd0;
+$chr-leave-calendar-item-requested-border-height: 0.5em;
+$chr-leave-calendar-legend-size:                  20px;
+
+.chr_leave-calendar {
+
+  @media (min-width: $screen-lg-min) {
+    table-layout: fixed;
+  }
+}
+
+
+.chr_leave-calendar__days {
+
+  > th {
+    font-size: 10px;
+    padding: 10px !important;
+    text-align: center;
+
+    @media (min-width: $screen-lg-min) {
+      padding: 10px 0 !important;
+    }
+
+    > span {
+      display: block;
+    }
+  }
+}
+
+
+.chr_leave-calendar__items {
+
+  > td {
+    overflow: hidden;
+    height: $chr-leave-calendar-cell-height;
+    padding: 0 !important;
+  }
+}
+
+
+.chr_leave-calendar__item {
+  color: $light;
+  display: block;
+  font-size: $chr-leave-calendar-cell-font-size;
+  height: 100%;
+  position: relative;
+  text-align: center;
+  text-transform: uppercase;
+  width: 100%;
+
+  > span {
+    bottom: 0;
+    height: 1.2em;
+    left: 0;
+    margin: auto;
+    position: absolute;
+    right: 0;
+    top: 0;
+  }
+}
+
+.chr_leave-calendar__item--alt {
+  background: transparent !important;
+  border: 1px solid transparent;
+  color: $gray-darker;
+}
+
+.chr_leave-calendar__item--requested {
+
+  &:before {
+    background: $chr-leave-calendar-item-requested-border-color;
+    content: '';
+    display: block;
+    height: $chr-leave-calendar-item-requested-border-height;
+    position: absolute;
+    width: 100%;
+    top: 0;
+  }
+}
+
+
+.chr_leave-calendar__legend {
+
+  > .row:not(:first-child) {
+    margin-top: 20px;
+  }
+
+  .chr_leave-calendar__legend__entry {
+    margin-top: 10px;
+
+    @media (min-width: $screen-sm-min) {
+      margin-top: 0;
+    }
+  }
+}
+
+.chr_leave-calendar__legend__entry {
+  height: $chr-leave-calendar-legend-size;
+  overflow: auto;
+
+  & + & {
+    margin-top: 10px;
+  }
+
+  > .chr_leave-calendar__item {
+    float: left;
+    font-size: 8px;
+    margin-right: 10px;
+    width: $chr-leave-calendar-legend-size;
+  }
+}
+
+.chr_leave-calendar__legend__label {
+  font-size: 12px;
```

Comment:
I mean to create variables for font-sizes, example
```
$header-font: 20px;
$label-font: 12px;
```

So that all labels, header would have same font-size;
For now we can start with $label-font and implement this gradually in future PR's.